exp-models/addon/models/session.js

```diff
@@ -18,6 +18,9 @@ export default DS.Model.extend(AnonJamModel, {
     experimentId: DS.attr('string'),
     experimentVersion: DS.attr('string'),  // TODO: Currently this field is not acted on in any way
 
+    frameIndex: DS.attr(),
```

Comment:
These new model attributes appear to be ISP-specific fields. Can you elaborate on why these properties are in the shared model in `exp-addons`, rather than in a customized subclass of this model in the ISP repo?

(You can extend the base model using the same technique you used for the authenticators early this week- import and extend)